mmv1/third_party/terraform/services/compute/resource_compute_subnetwork_test.go.tmpl

```diff
@@ -60,26 +61,30 @@ func TestAccComputeSubnetwork_basic(t *testing.T) {
 
 	var subnetwork1 compute.Subnetwork
 	var subnetwork2 compute.Subnetwork
+        var subnetwork3 compute.Subnetwork
```

Comment:
If you're using a separate test, all the subnetwork4 / `network-with-migration-purpose` stuff needs to be removed from this function